.github/workflows/ci-asp-app.yaml

```diff
@@ -0,0 +1,133 @@
+
+name: CI-ASP
+
+# ----------------------------------------------------------------------------------------------------------
+# CI Pipeline for building ASP app and deplyoing it to Azure App Service
+# - get config file via Rattus https://github.com/Saritasa/rattus
+# - deploy to Azure App Service
+# 
+# Original Author: Rostislav Udaltsov
+# Created: 05/11/2023
+# ----------------------------------------------------------------------------------------------------------
+
+on:
+  workflow_call:
+    inputs:
+      environment:
```

Comment:
- use lowercase for all vars.
- add description to each and every value.

Reply:
done